Packages/com.unity.render-pipelines.high-definition/Editor/Upgraders/UpgradeStandardShaderMaterials.cs

```diff
@@ -12,7 +12,7 @@ static List<MaterialUpgrader> GetHDUpgraders()
             var upgraders = new List<MaterialUpgrader>();
             upgraders.Add(new StandardsToHDLitMaterialUpgrader("Standard", "HDRP/Lit"));
             upgraders.Add(new StandardsToHDLitMaterialUpgrader("Standard (Specular setup)", "HDRP/Lit"));
-            upgraders.Add(new StandardsToHDLitMaterialUpgrader("Standard (Roughness setup)", "HDRP/Lit"));
+            upgraders.Add(new StandardsToHDLitMaterialUpgrader("Autodesk Interactive", "HDRP/Lit"));
```

Comment:
Hey @RemyUnity could you confirm this? And also I think we need to update Standard (Specular setup)